install.sh

```diff
@@ -0,0 +1,5 @@
+conda env create --file environment.yml
```

Comment:
Really nice idea to keep the installation simple for users. I should add this to my repos as well!